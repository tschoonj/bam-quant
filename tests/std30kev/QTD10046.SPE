$DATA:
0 2000
15242
10392
4935
1835
563
142
50
20
14
13
9
11
6
1
6
24
242
2629
4026
4040
3930
3642
3659
3616
3533
3436
3358
3148
3129
3077
3004
2910
2989
2791
2743
2714
2731
2575
2535
2485
2445
2479
2439
2451
2393
2423
2350
2549
2436
2439
2395
2530
2464
2472
2524
2564
2615
2597
2611
2671
2632
2697
2720
2686
2792
2739
2824
2789
2817
2694
2868
2760
2816
2621
2696
2730
2607
2530
2321
2333
2237
2216
2038
2049
2020
1794
1845
1712
1664
1705
1642
1591
1629
1643
1540
1379
1416
1502
1482
1439
1500
1487
1422
1422
1317
1257
1202
1202
1094
1076
1033
975
909
866
939
806
862
850
781
797
779
764
754
713
698
719
729
696
651
669
701
632
633
599
584
608
608
610
562
610
596
570
564
574
540
567
533
528
569
533
476
510
506
537
486
466
531
499
492
521
491
475
448
462
475
487
472
460
497
467
434
430
380
432
412
408
440
430
435
402
419
430
453
439
420
438
425
426
471
362
386
382
381
385
420
372
412
372
392
398
371
407
419
389
354
394
386
390
357
378
383
366
347
377
411
366
358
393
360
337
350
380
372
363
376
378
367
314
334
367
345
368
322
339
341
382
366
322
348
340
324
366
382
326
331
318
301
311
370
338
338
326
353
327
294
324
351
306
324
338
307
311
326
365
378
375
401
420
474
420
435
352
375
334
326
351
308
319
283
336
288
312
294
291
337
324
330
318
287
312
295
334
358
364
361
338
281
334
303
276
302
291
282
299
307
317
296
306
302
324
346
348
424
446
570
599
746
727
644
656
538
499
426
363
333
332
296
315
322
274
283
296
291
262
289
259
311
329
295
268
274
287
275
321
296
344
323
327
350
347
327
366
301
307
303
309
285
267
301
265
299
293
257
280
293
290
292
300
302
296
318
300
286
324
280
272
258
268
277
275
241
281
284
295
246
267
282
268
265
242
280
273
261
294
270
299
280
289
279
320
284
288
279
298
276
249
269
282
305
289
283
255
280
260
294
254
291
264
256
279
270
256
286
282
264
265
263
273
288
260
281
263
249
268
277
236
284
280
284
275
269
236
288
292
267
267
271
253
272
271
229
277
288
291
283
262
311
301
306
291
299
279
271
279
243
257
288
251
252
254
261
276
225
260
254
265
254
239
282
287
269
257
234
262
249
225
238
262
278
280
286
246
265
248
241
265
241
233
264
245
254
239
255
261
257
251
271
244
240
250
276
245
271
268
283
263
244
258
268
247
256
237
263
263
255
229
264
262
262
271
277
261
237
231
254
263
248
273
251
260
230
292
239
248
270
260
257
262
269
254
267
270
228
242
233
248
245
248
229
251
256
256
217
237
247
243
271
251
257
245
259
248
253
232
275
260
257
237
238
253
234
265
245
247
228
235
260
253
223
259
241
254
233
229
271
237
238
247
235
242
248
257
234
247
254
219
220
252
247
257
237
231
240
254
261
247
253
259
278
264
237
277
264
306
268
283
249
283
266
256
280
297
266
249
233
236
268
265
254
262
244
290
267
272
293
291
266
275
278
273
279
260
287
222
261
264
249
229
283
228
248
300
257
274
254
253
247
248
277
266
261
241
246
270
267
257
250
257
255
288
248
242
257
234
254
264
279
280
243
269
260
272
246
280
252
274
263
239
295
273
258
269
263
263
264
284
280
264
300
263
253
261
255
260
296
281
268
290
268
306
258
265
320
302
300
301
304
348
329
313
330
352
314
329
311
302
333
335
333
360
348
308
332
314
325
386
352
363
342
338
371
337
380
336
363
364
373
344
381
389
377
391
402
415
464
582
690
931
1253
1702
2365
3267
4440
5777
7403
9149
10584
12281
13744
14170
14456
14086
13373
12063
10416
8489
6634
5185
3804
2696
1876
1367
973
688
545
471
423
419
381
363
371
367
309
342
333
355
357
302
352
337
356
337
334
327
363
343
371
388
355
354
367
363
360
358
348
345
344
371
367
376
382
360
354
403
371
375
393
403
360
392
411
373
395
411
384
401
391
412
396
409
432
439
452
448
429
441
423
421
449
449
506
491
554
562
623
756
864
1055
1206
1659
1773
2007
2299
2586
2566
2683
2525
2336
2066
1930
1622
1391
1097
1009
909
827
823
760
787
743
773
754
715
674
654
620
562
519
522
436
463
485
493
479
450
486
517
482
483
524
484
551
519
534
533
569
538
602
533
567
548
549
568
600
609
570
626
640
642
600
659
654
665
648
687
715
759
738
712
809
765
813
840
830
821
862
930
999
964
999
1151
1148
1146
1243
1270
1297
1389
1422
1440
1505
1539
1584
1626
1700
1619
1780
1773
1729
1788
1755
1770
1804
1799
1831
1878
1804
1844
1919
1889
1907
1979
1956
1954
1909
1965
1979
1948
2005
2022
1957
1992
2044
1957
1962
1964
1980
1987
2057
1987
2040
2022
1983
2029
2018
2141
2099
2208
2152
2228
2260
2327
2321
2415
2490
2571
2621
2756
2803
2936
3183
3297
3387
3662
4016
4647
5330
6580
8393
11273
15705
21942
30754
42237
57571
75790
97868
122618
150032
178704
209148
237959
265950
293966
317557
339233
355492
367407
369580
364077
348804
326312
294915
256974
217854
178305
141237
107968
79747
56863
39288
26947
17938
12072
8156
5723
4051
3145
2497
2193
1833
1701
1551
1410
1344
1327
1224
1147
1170
1037
1079
1032
972
982
860
946
895
876
934
916
837
858
869
849
832
865
844
881
834
761
741
725
776
737
699
779
713
738
743
693
637
675
692
759
672
681
706
674
642
666
709
683
659
710
709
637
670
726
676
627
737
722
706
714
751
727
749
742
786
789
769
771
824
833
855
823
820
858
898
895
901
943
956
999
976
1000
997
1096
1146
1245
1283
1446
1670
1865
2388
2960
3955
5510
7663
10298
14256
19338
25115
32186
39620
47513
54665
61037
67017
69627
70641
69265
65359
60109
53130
45324
37410
30292
24016
18107
13712
10190
7909
6453
5690
5432
5939
6500
7487
8650
9533
10425
11467
11927
12217
11842
11123
9963
8862
7637
6297
5016
3997
2939
2135
1521
1082
731
460
309
225
171
131
88
91
90
63
64
68
62
45
52
59
56
48
46
53
44
35
54
48
48
54
40
51
50
33
33
37
35
31
46
42
32
47
38
29
36
38
40
38
29
27
26
27
31
25
17
25
24
27
21
31
27
23
19
20
14
27
19
22
23
17
25
19
17
20
19
25
17
11
25
23
10
16
12
11
16
18
14
12
21
12
16
18
13
25
15
13
17
14
14
13
17
12
20
13
8
14
13
12
15
18
14
13
15
12
12
6
20
12
8
8
14
14
9
9
9
15
13
13
7
13
11
16
6
9
14
7
8
9
9
13
10
8
10
10
10
10
8
7
16
7
17
15
18
10
15
13
11
18
12
6
9
9
18
13
9
10
7
10
6
7
7
12
6
13
8
8
9
9
8
11
7
6
17
11
8
18
6
13
8
10
6
10
11
9
11
7
12
9
13
7
11
9
6
12
7
10
14
14
8
8
10
6
5
8
7
8
7
11
6
4
4
6
12
8
6
12
12
8
11
12
12
8
4
8
5
6
7
4
9
5
6
9
8
7
8
6
5
5
12
7
9
3
6
7
7
8
8
4
6
10
8
4
7
8
15
10
9
7
8
7
15
12
10
10
7
8
9
10
5
7
7
9
11
13
11
3
11
5
9
7
8
5
7
12
13
8
7
3
6
2
11
7
10
11
6
6
12
11
11
10
4
12
15
7
6
9
6
10
7
7
6
10
8
8
8
7
7
15
6
11
8
6
10
13
10
11
6
12
6
14
12
13
12
18
14
18
7
10
11
10
8
9
9
9
9
4
9
7
19
10
5
11
9
10
10
12
11
4
12
13
10
8
11
8
9
8
9
5
8
16
12
7
8
9
12
14
8
9
9
8
9
15
8
10
3
8
12
9
6
8
10
7
10
7
8
13
13
9
20
10
14
16
9
15
5
13
11
10
13
14
8
10
13
10
9
11
18
8
14
12
15
13
16
13
17
14
12
12
7
17
9
4
11
12
8
11
9
9
8
13
7
7
15
18
12
12
16
18
15
8
22
15
13
10
14
23
22
18
13
9
17
15
21
14
14
13
10
16
13
13
19
13
16
15
10
16
15
16
10
14
14
8
15
14
17
13
16
12
8
16
13
15
15
13
7
16
18
14
20
23
14
17
26
17
18
22
19
22
27
31
24
31
23
22
30
27
26
27
26
28
25
31
25
19
26
19
26
20
15
18
22
24
17
13
14
17
16
14
21
10
13
12
14
6
3
16
12
6
10
11
11
7
6
5
9
6
8
5
11
3
6
11
6
8
6
13
5
10
8
7
10
11
10
12
16
8
12
16
16
15
16
30
22
25
28
37
43
56
63
68
80
78
108
116
134
144
126
136
131
130
153
149
153
156
138
139
128
125
108
91
89
83
66
50
53
44
42
43
24
23
17
10
12
9
8
8
14
11
13
7
3
8
2
13
6
10
7
8
7
3
5
4
4
7
2
12
7
6
5
11
10
8
6
7
8
8
10
6
4
6
6
8
6
14
10
12
9
11
11
7
9
5
9
5
14
7
10
8
12
10
11
13
12
20
29
13
17
29
27
23
24
25
28
30
26
24
34
40
37
20
26
27
32
41
25
29
19
15
19
24
18
15
21
9
13
12
18
16
15
13
19
18
14
24
23
34
36
32
27
29
36
29
28
31
38
40
42
33
37
27
27
36
20
28
20
28
21
14
26
20
16
9
14
12
16
13
17
21
25
18
24
13
18
22
24
17
29
23
21
18
20
24
24
18
21
30
25
28
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10046.spe
Start: 22.07.2014 10:16:23
Stop: 22.07.2014 10:18:03
BESSY: 298.6054857 298.9768276
IONI: 1.957616E-11 1.956018E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 66672
Run: 100000
Dead: 33
ICR: 114520
OCR: 0
Accumulated BESSY: 59660.5256886
Accumulated Ioni: 3.91556795E-09
N Accumulation: 200
Normfactor:  7.6611109e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pd
 
exp_x 112.720397949219 112.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  46   1     21.123   6067434.      2462.      93.86
