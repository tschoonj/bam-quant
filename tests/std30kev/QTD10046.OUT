     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:52  บ   
     บ  Spectrum: QTD10046.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =   312.8   last change =   .06%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -24.5 ๑   100.0   บ     -24.5 ๑      .7   บ   
     บ  GAIN (eV/ch)  บ     20.202 ๑   2.000   บ    20.202 ๑    .001   บ   
     บ det NOISE (eV) บ      126.8 ๑    40.0   บ     126.8 ๑     3.9   บ   
     บ  FANO factor   บ       .108 ๑    .050   บ      .108 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Pd-Ka                           6067434. ๑   2462.           บ   
     บ    KA1        21.177    .65302     3962184. ๑   1608.    107.6  บ   
     บ             1049.460    254.72         818.           9.25E-01  บ   
     บ    KA2        21.020    .34698     2105251. ๑    855.     97.8  บ   
     บ             1041.688    254.01         844.           9.29E-01  บ   
     บ                                                                 บ   
     บ  2 Pd-Kb                           1144718. ๑   1070.           บ   
     บ    KB1        23.819    .56746      649580. ๑    608.     36.2  บ   
     บ             1180.237    266.40         526.           8.46E-01  บ   
     บ    KB3        23.791    .29171      333924. ๑    313.     37.8  บ   
     บ             1178.851    266.28         530.           8.47E-01  บ   
     บ    KB2        24.299    .14083      161214. ๑    152.     28.4  บ   
     บ             1203.997    268.47         477.           8.29E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
